common/api/presentation-components.api.md

```diff
@@ -167,10 +167,10 @@ export interface DataProvidersFactoryProps {
 export const DEFAULT_PROPERTY_GRID_RULESET: Ruleset;
 
 // @beta @deprecated
-export function DEPRECATED_controlledTreeWithFilteringSupport<P extends ControlledTreeWithVisibleNodesProps>(TreeComponent: React.FC<P>): React.FC<Pick<P & ControlledTreeWithFilteringSupportProps, "filter" | "onFilterApplied" | "onMatchesCounted" | "activeMatchIndex" | "nodeLoader" | "onNodeLoaderChanged" | Exclude<keyof P, "visibleNodes">>>;

```

Comment:
Revert this file. `rush extract-api` is terrible. I usually just run `npm run extract-api` in the directories of the packages I modified.

Reply:
Fixed. Thanks.   I usually discard the changes before I commit but running extract-api in directories sounds more reliable.

Reply:
And then somehow the merge re-added a bunch of changes.   I'll have to complete this next week.